types/block.go

```diff
@@ -1587,8 +1590,9 @@ func (data *EvidenceData) FromProto(eviData *tmproto.EvidenceList) error {
 	return nil
 }
 
-func (data *EvidenceData) splitIntoShares(shareSize int) NamespacedShares {
+func (data *EvidenceData) splitIntoShares() NamespacedShares {
 	shares := make([]NamespacedShare, 0)
+	rawDatas := make([][]byte, len(data.Evidence))
```

Comment:
You probably mean 
```suggestion
	rawDatas := make([][]byte, 0, len(data.Evidence))
```
here?